### PR TITLE
gofumpt

### PR DESCRIPTION
**Pull request by faddat**
I've been using gofumpt a lot and really like the regularity that it adds to code.

You should be able to exactly reproduce this pr like:

go install mvdan.cc/gofumpt@latest
gofumpt -w -l .

and it is probably not advisable to look at every single file but instead diffs.

gofumpt describes itself as a stricter, `go fmt` compatible version of `go fmt`

https://github.com/mvdan/gofumpt

PS if it's not desired or useful please feel free to close this PR

if it is desired/useful, it can be ci enforced if we'd like it to be

I think Osmosis is going with "once per release"

### CODE REVIEW

**Comment by jaekwon** [comment]
Wow that's interesting... can this be done without the bak folder please?

**Comment by moul** [comment]
Btw @jaekwon, what’s the plan for the bak folder?

**Comment by faddat** [comment]
We can do this without the bak folder.

**Comment by faddat** [comment]
all set :)

File: pkgs/crypto/keys/keybase.go [hunk 1]
```diff
@@ -308,7 +308,8 @@ func (kb dbKeybase) ExportPubKey(nameOrBech32 string) (astr string, err error) {
 // ExportPrivKey returns a private key in ASCII armored format.
 // It returns an error if the key does not exist or a wrong encryption passphrase is supplied.
 func (kb dbKeybase) ExportPrivKey(name string, decryptPassphrase string,
-	encryptPassphrase string) (astr string, err error) {
+	encryptPassphrase string,
```
Comment (moul) [inline pkgs/crypto/keys/keybase.go:311]: not a big fan of this one, but okay

**Comment by moul** [comment]
Hey @faddat, sorry, but I wanted to push a fix on your PR, but you probably didn't clicked on "allow maintainers to make changes on my PR"

I created a new PR and updated the body to say that you were the original author (https://github.com/gnolang/gno/commit/bcfe472371a9d2a1dc188ccf48118261a9bb808c), but git thinks that it's me, I'm sorry for taking the authorship of your work.

Please, make sure to check the checkbox for the next PRs, and on my side, I'll try to use the comment system a little bit more (here it was difficult, because the GitHub's comment system works well for files with diff, but not for newly added files, and especially binaries).

**Pull request closed without merging**